templates/download.sh.erb

```diff
@@ -13,12 +13,14 @@ function check_exec {
 DOWNLOAD_URL="<%= scope.lookupvar('vmwaretools::archive_url') %>/VMwareTools-<%= scope.lookupvar('vmwaretools::version') %>.tar.gz"
 DOWNLOAD_LOCATION="<%= scope.lookupvar('vmwaretools::working_dir') %>/VMwareTools-<%= scope.lookupvar('vmwaretools::version') %>.tar.gz"
 GOOD_CHECKSUM="<%= scope.lookupvar('vmwaretools::archive_md5') %>"
+CURL_PROXY="<%= scope.lookupvar('vmwaretools::curl_proxy') %>"
```

Comment:
You shouldn't need to declare this variable, you can just change L22 to use `scope.lookupvar` for the `-x` parameter.